mapiproxy/modules/mpm_cache.c

```diff
@@ -386,14 +387,14 @@ static NTSTATUS cache_pull_OpenMessage(struct dcesrv_call_state *dce_call,
 
 	message = talloc((TALLOC_CTX *)mpm, struct mpm_message);
 	NT_STATUS_HAVE_NO_MEMORY(message);
-	
-	message->session = mpm_session_init((TALLOC_CTX *)mpm, dce_call);
+
+	message->session = mpm_session_init(dce_call, NULL);
 	NT_STATUS_HAVE_NO_MEMORY(message->session);
```

Comment:
This call and all the other mpm_session_init calls in this file are going to cause openchange to crash.

You specify a struct GUID * pointer being NULL and there is no check done anywhere before assignment at https://github.com/openchange/openchange/pull/389/files#diff-1acfc358c59edcd26a813334c163524cR59